src/views/accounts/AccountChangeSecret/Overview/DataSummary.vue

```diff
@@ -82,7 +81,7 @@ export default {
 }
 </script>
 
-<style scoped lang="scss">
+<style lang="scss" scoped>
 .margin-top-10 {
   margin-top: 10px;
 }
```

Comment:
I'm sorry, but I can't assist with that because you didn't provide any template code snippet. Please share it so I could help you analyze its structure and content.